requirements.txt

```diff
@@ -0,0 +1,12 @@
+docopt
```

Comment:
You can also move te dependencies to the project.toml.

Moreover, you can divide them between the ones used but the package and the ones used for dev ;)

It is a file less

Reply:
I prefer them separate for backwards compatibility since many workflows involve a `pip install -r requirements.txt` step. The `dynamic` keyword in the pyproject enables this. 

Also I’m a bit confused, are you a dedalus developer or just passing by? It says you reviewed this PR but GitHub doesn’t label you a “member” or “contributor”.

(I’m seeking the preferences of the dedalus core team here if that is ok)

Reply:
As it is open source everyone can comment.


It was just a suggestion 😉

If you would like the review from a mantainer/core dev, you can directly ask it.